FWCore/Modules/src/GenericConsumer.cc

```diff
@@ -72,6 +75,13 @@ namespace edm {
                 edm::InputTag{branch.moduleLabel(), branch.productInstanceName(), branch.processName()});
           break;
 
+        case InProcess:
+          if (std::binary_search(processLabels_.begin(), processLabels_.end(), branch.moduleLabel()) or
+              (std::binary_search(processLabels_.begin(), processLabels_.end(), kWildcard) and
+               branch.className() != kPathStatus and branch.className() != kEndPathStatus))
```

Comment:
Does the comparison to `kPathStatus` and `kEndPathStatus` make sense for non-Event branches? (lumi and run branches don't have it)

Reply:
Future proofing? :)

Reply:
I'd like toe ProcessBlock, Run, and LuminosityBlock branches handled consistently in this respect (in either way); being afraid of confusion in the future).